test/box/net.box_iproto_id_gh-6253.result

```diff
@@ -0,0 +1,276 @@
+-- test-run result file version 2
```

Comment:
For features AFAIK we do not specify issue number in the test file name. It applies only to bugfix tests.

Reply:
I don't think there's such a rule. E.g. `test/box/net.box_timeout-gh-3107.test.lua` tests a feature (async timeout), but still appends a ticket number. I think a ticket number doesn't hurt so let's leave it as is.

Reply:
Look at SOP, section "Writing tests":
```
Every patch should have a test case. 

When writing tests for a feature, set some time apart to design a test plan. A good test plan lists all corner cases, takes into account how a new feature could interact with existing ones.

Select a proper location for your test: choose tap or result/reject style suite, depending on your preferences, and choose a suite according to the domain you’re working on.

We do not create a separate file for each feature - that would lead to a huge number of files.

For a bug fix, please create a separate file with an autonomous test case in a corresponding test suite. Name should follow this format:
<suite name>/gh-XXX-two-three-word-title
```
So for features we do not create a new file unless it is something big and independent, and the rule of `gh-####` applies only to bugfixes. The test you are referring to was made and pushed without a proper review, so we have a little inconsistency there.

The same can be found here: https://github.com/tarantool/tarantool/wiki/Code-review-procedure#testing.

Reply:
Right, renamed.